example_module_mailtheme/views/templates/admin/index.html.twig

```diff
@@ -1,4 +1,7 @@
-{% extends '@PrestaShop/Admin/layout.html.twig' %}
+{% extends '@!PrestaShop/Admin/layout.html.twig' %}
```

Comment:
I don't remember, what is the interest of the initial `!` ?

Reply:
It's to be sure we use PrestaShop's template and not one from a module as it could lead to a infinite loop